frontend/packages/dev-console/src/components/EmptyState.tsx

```diff
@@ -3,22 +3,28 @@ import { useTranslation } from 'react-i18next';
 import { connect } from 'react-redux';
 import { Gallery, GalleryItem } from '@patternfly/react-core';
 import { CatalogTile } from '@patternfly/react-catalog-view-extension';
-import { history, useAccessReview } from '@console/internal/components/utils';
-import { useExtensions } from '@console/plugin-sdk';
+import { history, LoadingBox, useAccessReview } from '@console/internal/components/utils';
 import { RootState } from '@console/internal/redux';
 import { ALL_NAMESPACES_KEY, PageLayout } from '@console/shared';
 import { HIDE_QUICK_START_ADD_TILE_STORAGE_KEY } from '@console/shared/src/components/quick-starts/quick-starts-catalog-card-constants';
 import QuickStartsLoader from '@console/app/src/components/quick-starts/loader/QuickStartsLoader';
 import QuickStartsCatalogCard from '@console/shared/src/components/quick-starts/QuickStartsCatalogCard';
-import { isAddAction, AddAction } from '../extensions/add-actions';
+import {
+  ResolvedExtension,
+  useResolvedExtensions,
+} from '@console/dynamic-plugin-sdk/src/api/useResolvedExtensions';
+import {
+  isAddAction,
+  ResolvedAddAction,
+} from '@console/dynamic-plugin-sdk/src/extensions/add-actions';
 
 const navigateTo = (e: React.SyntheticEvent, url: string) => {
   history.push(url);
   e.preventDefault();
 };
 
 interface ItemProps {
-  action: AddAction;
+  action: ResolvedExtension<ResolvedAddAction>;
```

Comment:
Can't we just use `ResolvedAddAction` here?

Reply:
Since `useResolvedExtensions` returns `ResolvedExtension<E>[]` we would need to use `ResolvedExtension` type here.